flake.nix

```diff
@@ -87,6 +87,10 @@
                                 # Uncomment to use a custom database URL
                                 # DATABASE_URL = "postgresql://postgres:...CHANGE-ME";
 
+                                # Uncomment to use a custom session secret, ensuring sessions aren't invalidated
+                                # on each deploy.
+                                # IHP_SESSION_SECRET = "CHANGE-ME";
```

Comment:
Should we link to a service like https://it-tools.tech/token-generator , for creating the token?

Reply:
I think the session key is a base64 encoded private key, therefore that won't work

Reply:
I'll quickly hack a binary that produces a hash on the CLI